lib/dom-actions.ts

```diff
@@ -76,7 +76,7 @@ export class DomActions implements DomActionsProvider {
         return document.cookie.includes(substring);
     }
 
-    hide(selector: string, method: HideMethod): boolean {
+    hide(selector: string, method?: HideMethod): boolean {
```

Comment:
Is it dangerous that developers using `hide` might accidentally choose the wrong `HideMethod` now that it’s not required?  (It being required forces one to stop and think.)

Reply:
Same here, this just codifies the existing default. I'm not sure actually how we ended up here, I suspect Typescript became stricter after some update.